apps/ag-grid-storybook/src/app/ag-grid/ag-grid-stories.component.stories.ts

```diff
@@ -0,0 +1,27 @@
+import { Meta, Story, moduleMetadata } from '@storybook/angular';
```

Comment:
the file pattern of these stories seems different than all the others...is that intentional? Others are like `ag-grid.component.ts` and `ag-grid.component.stories.ts`

Reply:
The name was to avoid duplicate `AgGridModule` definitions.